docs/root/start/simple_example.rst

```diff
@@ -0,0 +1,239 @@
+.. _start_simple:
+
+Simple Example
+==============
+
+This is perhaps the most minimal Envoy configuration which can be used to validate basic plain HTTP
+proxying is available in :repo:`configs/google_com_proxy.v2.yaml`. This is not
+intended to represent a realistic Envoy deployment.
+
+Quick Start
+-----------
+
+This Quick Start runs the example quickly using files in the Envoy repo. The sections below explain
```

Comment:
`This quick start gets you started with only the files in the Envoy repo.` (Or some other rewording, I don't like "runs the example quickly", as that can be read as having it run fast on the server, which is not what we want).